### DISCUSSION TITLE
Laravel integration for tabler

### DISCUSSION THREAD

**Discussion by Nejcc** [tabler/tabler#1031] [Show and tell]
I am preparing laravel version of tabler with full-features startup kit  

- authentication
- user and roles permissions
- notifications
- subscription
- plan / pricing
- dashboard
- and more... 

if you have wishlist please let me known

**Comment by jappi00** [comment]
Do you want to use Livewire? Sounds interesting!

**Reply by Nejcc** [reply]
No i will use component based it's much easier then to implement into vue or react @jappi00

**Comment by takielias** [comment]
This is the Latest Laravel Admin Dashboard Scaffolding Package with built-in Auth https://github.com/takielias/tablar
